src/main/java/mat/server/util/MeasureUtility.java

```diff
@@ -69,6 +69,22 @@ public static String formatVersionText(String revisionNumber, String version) {
         return formatVersionText(version) + "." + revisionFormat.format(Integer.parseInt(revisionNumber));
     }
 
+    public static String formatVersionTextForIncludeLibraries(String version) {
+        StringUtility su = new StringUtility();
+        String[] versionArr = version.split("\\.");
+        String majorVersion = "", minorVersion = "", revisionNumber = "";

```

Comment:
I believe most of the code style checkers recommend to define each variable on a separate line.

Reply:
Updated it